index.php

```diff
@@ -132,15 +140,47 @@
             $url = sprintf('team/%s', $platform);
             break;
     }
+    return $verifyUrl($url) ? $routes[$url] : $routes['default'];
+};
+
+$langUrlBuilder = function ($url, $platform) use ($verifyUrl, $routes) {
+    if ($verifyUrl($url)) {
+        return $routes[$url] . "/" . $platform;
+    } else {
+        return $routes['default'];
+    }
+};
+
+$langs = ['ar', 'bg', 'bs', 'cs', 'de', 'el', 'en', 'es', 'fr', 'hr', 'hu', 'id', 'ja', 'pl', 'pt', 'ro', 'ru', 'tr', 'uk', 'vi', 'zh'];
 
```

Comment:
I think we can do this with the crowdin API. Get the available languages, cache them and voila. Less things to remember to update :)